lsif/src/dump-manager/settings.ts

```diff
@@ -18,3 +18,15 @@ export const DOCUMENT_CACHE_CAPACITY = readEnvInt('DOCUMENT_CACHE_CAPACITY', 102
 
 /** The maximum number of result chunks that can be held in memory at once. */
 export const RESULT_CHUNK_CACHE_CAPACITY = readEnvInt('RESULT_CHUNK_CACHE_CAPACITY', 1024 * 1024 * 1024)
+
+/** The maximum rate that the server will send upload payloads. */
+export const MAXIMUM_SERVE_BITS_PER_SECOND = readEnvInt('MAXIMUM_SERVE_BITS_PER_SECOND', 1024 * 1024 * 1024 * 10)
+
+/** The maximum chunksize the server will use to send upload payloads. */
+export const MAXIMUM_SERVE_CHUNKSIZE = readEnvInt('MAXIMUM_SERVE_CHUNKSIZE', 1024 * 1024 * 10)
```

Comment:
```suggestion
/** The maximum chunksize the server will use to send upload payloads, in bytes. */
export const MAXIMUM_SERVE_CHUNKSIZE = readEnvInt('MAXIMUM_SERVE_CHUNKSIZE', 1024 * 1024 * 10)  // 10 MiB
```

Or at least I'm assuming this is in bytes. We could also put the units in the name: `MAX_SERVE_CHUNK_BYTES` or words to that effect. YMMV.

Reply:
Updating.